HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentPrimaryButton.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  PaymentPrimaryButton.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import UIKit
+import GiniHealthAPILibrary
+
+final class PaymentPrimaryButton: UIView {
+    
+    private var giniHealthConfiguration = GiniHealthConfiguration.shared
```

Comment:
[Clatification]:  do we use `giniHealthConfiguration` here?

Reply:
yes, for the button font: `giniHealthConfiguration.textStyleFonts[.button]`